cmake/DD4hepBuild.cmake

```diff
@@ -652,11 +652,7 @@ macro(DD4HEP_SETUP_ROOT_TARGETS)
   SET(Python_FIND_FRAMEWORK LAST)
   IF((TARGET ROOT::PyROOT OR TARGET ROOT::ROOTTPython) AND ${ROOT_VERSION} VERSION_GREATER_EQUAL 6.19)
     # cmake 3.17.1 doesn't include python patch level in PYTHON_VERSION
-    IF(CMAKE_VERSION VERSION_EQUAL 3.17.1)
-      string(REGEX MATCH [23]\.[0-9]+ REQUIRE_PYTHON_VERSION ${ROOT_PYTHON_VERSION})
-    ELSE()
-      SET(REQUIRE_PYTHON_VERSION ${ROOT_PYTHON_VERSION})
-    ENDIF()
+    string(REGEX MATCH [23]\.[0-9]+ REQUIRE_PYTHON_VERSION ${ROOT_PYTHON_VERSION})
     dd4hep_debug("D++> Python version used for building ROOT ${ROOT_PYTHON_VERSION}" )
     dd4hep_debug("D++> Required python version ${REQUIRE_PYTHON_VERSION}")
     FIND_PACKAGE(Python ${ROOT_PYTHON_VERSION} EXACT REQUIRED COMPONENTS Development)
```

Comment:
Shouldn't it be here `REQUIRE_PYTHON_VERSION`

Reply:
yes